### ISSUE TITLE
Steps before transfer to sourcecred and release as action

### ISSUE THREAD

**Issue by vsoch** [vsoch/cred-action#7]
@Beanow I'm creating this for you and I to keep track of TODOs

 - [ ] Review and merge the update to not run the command twice, require project and project file #6 
 - [ ] Once this is done, @wchargin can rebase his branch, and #4  can finish review to be merged into master. We'd want to update that branch and retest here https://github.com/vsoch/sourcecred-cred/pull/9
 - [ ] Then we want to add the cache, this is essential! @Beanow you've done this in your branch, so we can advise from there. 
 - [ ] I don't think [this issue](https://github.com/vsoch/cred-action/issues/5) is essential for this first go, thoughts?

And then finally, if it's desired, we can move to sourcecred/cred-action and release as a GitHub action. I am being careful with my words and saying "if it's desired" because last time I was told that I was pushy about it, which is definitely not my intention! :/

**Comment by Beanow** [comment]
>  I don't think this issue (#5) is essential for this first go, thoughts?

Agreed, we can leave that for a later enhancement :]

**Comment by Beanow** [comment]
Created #8 to track the caching issue.

**Comment by Beanow** [comment]
Just want to ping and see if this plan is up to date.

What's left? And how can I help? :smile:

**Comment by vsoch** [comment]
I can’t do anything until @wchargin rebases and finishes his contribution. If I knew something to do trust me I’d either be doing it or telling you about it!
